.github/workflows/ansible-test-roles.yml

```
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         proxysql:
-          - 2.0.12
+          - 2.3.2
```

```suggestion
          - 2.3.2
```
I would suggest having at least the oldest version of proxysql supported by the collection and the latest one there (no need to add now, just an idea)

---

That's something we can place/add to this issue https://github.com/ansible-collections/community.proxysql/issues/75 :)